nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -93,9 +90,7 @@
    "outputs": [],
    "source": [
     "samples = df.sample(frac=1e-4)\n",
-    "scatter = hv.Scatter(samples, 'trip_distance', 'fare_amount')\n",
-    "labelled = scatter.redim.label(trip_distance=\"Distance, miles\", fare_amount=\"Fare, $\")\n",
-    "labelled.redim.range(trip_distance=(0, 20), fare_amount=(0,40)).opts(size=5)"
+    "samples.hvplot.scatter('trip_distance', 'fare_amount', xlabel='Distance, miles', ylabel='Fare, $', s=5, xlim=(0,20), ylim=(0,40))"
```

Comment:
Just a very quick comment, we should try to avoid long line lengths as it means users will have to scroll horizontally which isn't great. That's what I see on my desktop screen, and it would be worse on a mobile phone or tablet.

![image](https://github.com/holoviz-topics/examples/assets/35924738/5745e21e-aa8c-4665-a988-e8ac01c17c39)